lyncs_quda/dirac.py

```diff
@@ -369,8 +370,12 @@ def __call__(self, rhs, out=None):
         if self.dirac.full:
             self.quda(out.quda_field, rhs.quda_field)
         elif self.dirac.even:
+            if self.dirac.clover is not None:
+                self.dirac.clover.inverse_field
             self.quda(out.quda_field.Even(), rhs.quda_field.Even())
```

Comment:
Similarly here, the calculation of `clover.inverse_field` shouldn't be done in `self.quda`?